webapp/src/components/hiddenCardCount/hiddenCardCount.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+import {useIntl} from 'react-intl'
+
+import Button from '../../widgets/buttons/button'
+
+import {Card} from '../../blocks/card'
+import './hiddenCardCount.scss'
+
+type Props = {
+    hiddenCards: Card[]
+}
+
+const HiddenCardCount = (props: Props): JSX.Element => {
+    const intl = useIntl()
+    return (
+        <div className='HiddenCardCount'>
+            <div className='hidden-card-title'>{intl.formatMessage({id: 'limitedCard.title', defaultMessage: 'Cards Hidden'})}</div>
```

Comment:
looks like you'll want to run `npm run i18n-extract` to update the `en.json` file

I'd recommend updating your branch with `release-7.0` before you do that as my PR that was merged yesterday updated the `npm run i18n-extract` command

Reply:
Okay, got it.